event_sale_reservation/tests/test_event_sale.py

```diff
@@ -99,7 +99,7 @@ def wizard_step_2(self, wizard1):
         # Get form from 2nd chained action
         action = multi_action["actions"][1]
         return Form(
-            self.env[action["res_model"]].with_context(action["context"]),
+            self.env[action["res_model"]].with_context(**action["context"]),
```

Comment:
In this case it doesn't matter, as there's no previous context

Reply:
This is only to avoid pre-commit error: "_Context overridden using dict. Better using kwargs..._"